### PR TITLE
[memorymodule] Add new port

### PR DESCRIPTION
**Pull request by myd7349**
This PR adds new port: https://github.com/fancycode/MemoryModule

### CODE REVIEW

**Comment by JackBoosY** [comment]
I have a PR #9430, maybe you can test it by the way?

**Comment by myd7349** [comment]
Hi! @JackBoosY Thanks! I will give it a try later.

**Comment by PhoebeHui** [comment]
@myd7349, it seems memorymodule:arm64-windows passed in CI testing, could you remove it from ci baseline file?

##[error] PASSING, REMOVE FROM FAIL LIST: memorymodule:arm64-windows (C:\vsts\_work\3\s\scripts\ci.baseline.txt

**Comment by myd7349** [comment]
Hi! @PhoebeHui Thanks for your reply. I have removed `arm64-windows` from the list.

File: ports/memorymodule/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    OPTIONS
+        -DTESTSUITE=OFF
+        ${FEATURE_OPTIONS}
+)
+
+vcpkg_build_cmake(TARGET MemoryModule)
+
+file(INSTALL ${SOURCE_PATH}/MemoryModule.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
+
+file(INSTALL ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/MemoryModule.lib DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
```
Comment (JackBoosY) [inline ports/memorymodule/portfile.cmake]: If you only build debug/release, there may be errors here. Can you add judgment conditions here?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/memorymodule/portfile.cmake (hunk 1 above)
Comment (myd7349) [reply ports/memorymodule/portfile.cmake]: Hi! @JackBoosY Thanks for your review. Is it possible to just trigger a `Debug` or `Release` build in `vcpkg`?

Comment (JackBoosY) [reply ports/memorymodule/portfile.cmake]: @myd7349 Nope, the pipeline will test both debug and release, but adding this change will reduce the problems that occur after setting `VCPKG_BUILD_TYPE`.

We will give a thorough solution to this issue in the future, but for now, we better distinguish them manually.

Comment (myd7349) [reply ports/memorymodule/portfile.cmake]: Thanks! That makes sense to me.

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**